src/app/articles-list-item/articles-list-item.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+import { Article } from '../aricles-mock';
+
+@Component({
+  selector: 'app-articles-list-item',
+  templateUrl: './articles-list-item.component.html',
+  styleUrls: ['./articles-list-item.component.scss'],
+})
+export class ArticlesListItemComponent {
+  @Input() article?: Article;
```

Comment:
De facto it is not an optional property. FOr this real-world application, I think we can use something like this

```
@Input() article!: Article;
```

In this case, we can avoid the checks like `if (this.article) {`